backend/globaleaks/settings.py

```diff
@@ -232,13 +244,17 @@ def load_cmdline_options(self):
         if self.cmdline_options.user and self.cmdline_options.group:
             self.user = self.cmdline_options.user
             self.group = self.cmdline_options.group
-            self.uid = pwd.getpwnam(self.cmdline_options.user).pw_uid
-            self.gid = grp.getgrnam(self.cmdline_options.group).gr_gid
+
+            if platform.system() != 'Windows':
+                self.uid = pwd.getpwnam(self.cmdline_options.user).pw_uid
+                self.gid = grp.getgrnam(self.cmdline_options.group).gr_gi
```

Comment:
@NCommander: minor defect spotted :)

gr_gi instead of gr_gid

going to release a 3.0.21 to fix current release